tools/ci_build/github/linux/docker/Dockerfile.ubuntu_gpu

```diff
@@ -4,7 +4,7 @@ ARG PYTHON_VERSION=3.6
 ARG BUILD_EXTR_PAR
 
 ADD scripts /tmp/scripts
-RUN /tmp/scripts/install_ubuntu.sh -p $PYTHON_VERSION && /tmp/scripts/install_deps.sh -p $PYTHON_VERSION -d gpu -x $BUILD_EXTR_PAR && rm -rf /tmp/scripts
+RUN /tmp/scripts/install_ubuntu.sh -p $PYTHON_VERSION && /tmp/scripts/install_deps.sh -p $PYTHON_VERSION -d gpu -x "$BUILD_EXTR_PAR" && rm -rf /tmp/scripts
```

Comment:
i did not actually verify this yet, but i think that it will have the effect of passing whatever is in BUILD_EXTR_PAR, even if it contains whitespace, as the value of the -x option. if it contains a double quote, i'm not sure how to handle that.

previously install_deps.sh did not recognize the -x option - i think it was working because the variable name it uses is also BUILD_EXTR_PAR, which is an environment variable.

Reply:
So "" is not needed, right? 
BUILD_EXTR_PAR was added to install_deps.sh in order to optionally install torch and later transformers. It was working without "".

Reply:
i think it is needed. a simple test:

```
$ cat test.sh
#!/bin/bash

while getopts x: parameter_Option
do case "${parameter_Option}"
in
x) BUILD_EXTR_PAR=${OPTARG};;
esac
done

echo "BUILD_EXTR_PAR = '${BUILD_EXTR_PAR}'"

$ ./test.sh -x foo bar
BUILD_EXTR_PAR = 'foo'

$ ./test.sh -x "foo bar"
BUILD_EXTR_PAR = 'foo bar'
```

previously, the "-x" wasn't being parsed by install_deps.sh because it was missing from the getopts option string.